$NetBSD: patch-bb,v 1.1 2010/07/31 21:09:14 dholland Exp $

Avoid conflict between configure script, which #defines u_int64_t on
the command line, and db.h, which provides it on platforms where it's
missing. PR 42805.

(Upstream ought to switch to using uint64_t and stdint.h.)

--- output.c.orig	2009-01-13 05:34:04.000000000 +0000
+++ output.c
@@ -35,6 +35,7 @@
 #include <sys/stat.h>
 #include <sys/utsname.h>
 #ifdef USE_DNS
+#undef u_int64_t
 #include <db.h>
 #endif
 
